packages/node/src/authenticatedFetch/fetchFactory.spec.ts

```diff
@@ -137,6 +137,25 @@ describe("buildBearerFetch", () => {
     );
   });
 
+  it("returns a fetch that calls the refresh token handler if appropriate", async () => {
+    const fetch = jest.requireMock("cross-fetch");
+    fetch.mockResolvedValue({ status: 401 });
+    const tokenSet = mockDefaultTokenSet();
+    tokenSet.refresh_token = "some refreshed refresh token";
```

Comment:
Just to make this slightly less confusing (also in other tests):

```suggestion
    tokenSet.refresh_token = "some rotated refresh token";
```

Reply:
That's clearer, but it's less funny to say ^^ 838255e114924476169ffe933a1cae2e20f908cf

Reply:
Can't argue with that :P